src/platform/packages/shared/kbn-monaco/src/languages/esql/language.ts

```diff
@@ -95,7 +95,7 @@ export const ESQLLang: CustomLangModuleType<ESQLCallbacks> = {
   },
   getSuggestionProvider: (callbacks?: ESQLCallbacks): monaco.languages.CompletionItemProvider => {
     return {
-      triggerCharacters: [',', '(', '=', ' ', '[', ''],
+      triggerCharacters: [',', '(', '=', ' ', '[', '', '?'],
```

Comment:
not you, but why do we have `''` as a trigger character??

Reply:
Also I think `,` is pretty much a useless trigger character because Monaco is always going to show no suggestions with a `,` prefix.

Anyway, something we can look at separately

Reply:
yeah if we can clean it up to not cause confusion would be great 🙌